libraries/standard/http/src/http_client.c

```diff
@@ -1155,18 +1388,17 @@ static HTTPStatus_t findHeaderInResponse( const uint8_t * pBuffer,
     HTTPStatus_t returnStatus = HTTP_SUCCESS;
     http_parser parser = { 0 };
     http_parser_settings parserSettings = { 0 };
-    findHeaderContext_t context =
-    {
-        .pField     = pField,
-        .fieldLen   = fieldLen,
-        .pValueLoc  = pValueLoc,
-        .pValueLen  = pValueLen,
-        .fieldFound = 0u,
-        .valueFound = 0u
-    };
+    findHeaderContext_t context = { 0 };
     size_t numOfBytesParsed = 0u;
 
-    /* Disable unused variable warning. */
+    context.pField = pField;
+    context.fieldLen = fieldLen;
+    context.pValueLoc = pValueLoc;
+    context.pValueLen = pValueLen;
+    context.fieldFound = 0u;
+    context.valueFound = 0u;
```

Comment:
It would be nice if these are placed right after `findHeaderContext_t context = {0}` but that's just my preference.

Reply:
Agreed, but to follow the FreeRTOS coding style, we keep all declarations at the topic of the function.
Besides, C90 raises a warning if the declarations and assignment expressions are not separated. Refer here: https://stackoverflow.com/questions/46942250/iso-c90-forbids-mixing-declarations-and-code